malformed/test_malformed_headers.py

```diff
@@ -0,0 +1,487 @@
+from framework import tester
+from helpers import tf_cfg, deproxy
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class MalformedRequestsTest(tester.TempestaTest):
+    backends = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'response' : 'static',
+            'response_content' :
+"""HTTP/1.1 200 OK
+Content-Length: 0
+Connection: close
+
+"""
+        },
+    ]
+
+    tempesta = {
+        'config' : """
+cache 0;
+listen 80;
+
+srv_group default {
+    server ${general_ip}:8000;
+}
+
+vhost default {
+    proxy_pass default;
+}
+""",
+    }
+
+    clients = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'addr' : "${tempesta_ip}",
+            'port' : '80'
+        },
+    ]
+
+    def common_check(self, request):
+        deproxy = self.get_server('deproxy')
+        deproxy.start()
+        self.start_tempesta()
+        self.assertTrue(deproxy.wait_for_connections(timeout=1))
+        deproxy = self.get_client('deproxy')
+        deproxy.start()
+        deproxy.make_request(request)
+        resp = deproxy.wait_for_response(timeout=5)
+        self.assertTrue(resp, "Response not received")
+        status = deproxy.last_response.status
+        self.assertEqual(int(status), 400, "Wrong status: %s" % status)
+
+    def test_accept(self):
+        # https://tools.ietf.org/html/rfc7231#section-5.3.2
+        request = 'GET / HTTP/1.1\r\n' \
+                  'Host: localhost\r\n' \
+                  'Accept: invalid\r\n' \
+                  '\r\n\r\n'
+        self.common_check(request)
+
+    def test_accept_charset(self):
+        # https://tools.ietf.org/html/rfc7231#section-5.3.3
+        # https://tools.ietf.org/html/rfc6365#section-2
+        request = 'GET / HTTP/1.1\r\n' \
+                  'Host: localhost\r\n' \
+                  'Accept-Charset: invalid\r\n' \
+                  '\r\n\r\n'
+        self.common_check(request)
+
+    def test_accept_encoding(self):
+        # https://tools.ietf.org/html/rfc7231#section-3.1.2.1
+        request = 'GET / HTTP/1.1\r\n' \
+                  'Host: localhost\r\n' \
+                  'Accept-Encoding: invalid\r\n' \
+                  '\r\n\r\n'
+        self.common_check(request)
+
+    def test_accept_language(self):
+        # https://tools.ietf.org/html/rfc4647#section-2.1
+        request = 'GET / HTTP/1.1\r\n' \
+                  'Host: localhost\r\n' \
+                  'Accept-Language: 123456789\r\n' \
+                  '\r\n\r\n'
+        self.common_check(request)
+
+    # Authorization
```

Comment:
Please add a TODO here or add the test.

Reply:
Authorization header can have arbitrary content. So it's nothing to test

Reply:
There are a lot of such comments, it's not clear whether it's TODO comments or there nothing to test. Since all well-known headers are listed here, need to explicitly write, that there is nothing to test. Just to keep others from thinking that the header is missed.